docs/RUNNING_ON_A_BUDGET.md

```
@@ -116,3 +129,46 @@ To prevent unnecessary API costs or hitting rate limits, implement the following
    ```bash
    npm run scan -- --verify
    ```
+
+---
+
+## 6. Worked Example: Running the Pipeline Cheaply
+
+Here is a concrete, end-to-end walkthrough of scanning for jobs and evaluating a single posting using **DeepSeek V3 via OpenRouter** and the standalone `openai-eval.mjs` evaluator. This bypasses the need for an expensive CLI agent for the heavy evaluation block.
+
+### Step 1: Scan for Job Offers (0 Tokens)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)